db/views/reporting_facility_states_v01.sql

```diff
@@ -0,0 +1,77 @@
+WITH
+    registered_patients AS (
+        SELECT registration_facility_region_id as region_id, month_date,
+               COUNT(*) as cumulative_registrations,
+               COUNT(*) FILTER (WHERE months_since_registration = 0) as monthly_registrations
+        FROM reporting_patient_states
+        WHERE hypertension = 'yes'
+        GROUP BY 1, 2
+    ),
+
+    assigned_patients AS (
+        SELECT assigned_facility_region_id as region_id, month_date,
+               COUNT(distinct(patient_id)) FILTER (WHERE htn_care_state = 'under_care') as under_care,
+               count(distinct(patient_id)) FILTER (WHERE htn_care_state = 'lost_to_follow_up') as lost_to_follow_up,
+               count(distinct(patient_id)) FILTER (WHERE htn_care_state = 'dead') as dead,
+               COUNT(*) as assigned_patients
+        FROM reporting_patient_states
+        WHERE hypertension = 'yes'
+        GROUP BY 1, 2
+    ),
+
+    treatment_outcomes_in_last_3_months AS (
+        SELECT assigned_facility_region_id as region_id, month_date,
+
+               count(distinct(patient_id)) FILTER (WHERE htn_treatment_outcome_in_last_3_months = 'controlled' AND htn_care_state = 'under_care') as controlled_under_care,
+               count(distinct(patient_id)) FILTER (WHERE htn_treatment_outcome_in_last_3_months = 'controlled' AND htn_care_state = 'lost_to_follow_up') as controlled_lost_to_follow_up,
```

Comment:
Chatted w/ @ssrihari about this and uncontrolled, and I think we agreed we should remove it, as its a non-sensical field to return.   LTFU is always going to be disjoint w/ controlled & uncontrolled, so it would be confusing to future devs or metabase users to leave it in.